notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -267,7 +267,7 @@
       },
```

Comment:
<p>Just a very small nit about the word "may" - it's can be vague, so guidance is to default to "might" or "can" depending on your meaning. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>